internal/client/main.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"google.golang.org/grpc/credentials/insecure"
+	"log"
+	"time"
+
+	pb "go_Playlist_gRPC/internal/proto/music_player"
+	"google.golang.org/grpc"
+)
+
+func main() {
+	conn, err := grpc.Dial("localhost:50051", grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		log.Fatalf("Failed to connect: %v", err)
+	}
+	defer conn.Close()
+
+	c := pb.NewMusicPlayerClient(conn)
+
+	// вызываем метод Play
+	//_, err = c.Play(context.Background(), &pb.Empty{})
+	//if err != nil {
+	//	log.Fatalf("Failed to play: %v", err)
+	//}
+	// вызываем метод AddSong
+	//ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	//defer cancel()
+	ans, err := c.AddSong(context.Background(), &pb.SongRequest{
```

Comment:
You can use the same context for each grpc operation. It's better to have tests for this.